src/components/form/experience/AddExperience.js

```diff
@@ -0,0 +1,100 @@
+import * as Yup from 'yup';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import React, { useContext } from 'react';
+
+import Button from '~/components/button/Button';
+import * as storage from '~/storage/LocalStorage';
+import { FormContext } from '~/components/FormContext';
+import InputText from '~/components/inputtext/InputText';
+import FormHeader from '~/components/formheader/FormHeader';
+import InputRadio from '~/components/inputradio/InputRadio';
+import OutsideClickDetector from '~/components/detector/OutsideClickDetector';
+
+const AddExperience = ({ onClose }) => {
+  const { data } = useContext(FormContext);
+
+  const validateExperience = Yup.object().shape({
```

Comment:
shall we move this to /validation folder?

Reply:
moved